CS/E1271/CheckMarkSelection.cs

```diff
@@ -1,19 +1,5 @@
-﻿// Developer Express Code Central Example:
-// Multiple selection using checkbox (web style)
-// 
-// The following example shows how to implement multiple selection in the web style
-// (via check boxes). The XtraGrid does not have this functionality built-in. The
-// GridCheckMarksSelection class allows you to implement it. End-users can
-// select/unselect rows, group rows or select/unselect all rows by clicking on the
-// column header. Changing check box value does not start row editing. This example
-// is based on the http://www.devexpress.com/scid=A371 article.
-// 
-// See
-// Also:
-// http://www.devexpress.com/scid=E990
-// 
-// You can find sample updates and versions for different programming languages here:
-// http://www.devexpress.com/example=E1271
+// Developer Express Code Central Example:
```

Comment:
Do we still call our Support Center examples "Code Central"? Or would it be better to say something like the following?

`// Developer Express Support Center Example:`

Reply:
```suggestion

```